include/compartment.h

```diff
@@ -29,11 +29,19 @@
 // about
 
 struct func_intercept;
+void compartment_transition_out();
```

Comment:
Let's start adding a doc comment to all function definitions as a habit. It makes my life as group idiot slightly less hard :)